api-report/map.api.md

```diff
@@ -260,7 +283,8 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
 export class SharedMap extends SharedObject<ISharedMapEvents> implements ISharedMap {
     [Symbol.iterator](): IterableIterator<[string, any]>;
     readonly [Symbol.toStringTag]: string;
-    constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
+    // @alpha
+    constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes, options?: IMapOptions);
```

Comment:
it's not reasonable to mark the map constructor alpha here. Since the option type is already marked alpha, I think you can just elide it (we will eventually want an options object in the API surface). You could leave an explicit note in the documentation that the options part of the constructor is currently in alpha if you want.